modules/account_pool_metrics_lambda.tf

```diff
@@ -0,0 +1,70 @@
+locals {
+  account_pool_metrics_count = var.account_pool_metrics_toggle == "true" ? 1 : 0
+}
+
+module "account_pool_metrics_lambda" {
```

Comment:
Not trying to let "perfect be the enemy or good" here, but it would be nice if the lambda also utilized the `count` parameter. That would require some changes to the lambda module and also might create wonkiness for the deploy / build scripts, so this is not a blocker or a requirement here.

Reply:
Yeah, agreed. I tried doing that initially, but it would have required a lot of changes.